core/tests/webdriverio_utils/StoryEditorPage.js

```diff
@@ -253,6 +265,11 @@ var StoryEditorPage = function() {
     }
   };
 
+  this.expectChapterStatusToBe = async function(index, status) {
+    var chapterTitles = await chapterStatusSelector();
+    expect(await chapterTitles[index].getText()).toEqual(status);
```

Comment:
Use `action.getText`.

I was expecting lint checks to catch this and report it! Looks like a regression, I'm looking into it!

Reply:
@Kej-r03 Is this resolved?